crates/sargon-uniffi/src/system/sargon_os/transactions/sargon_os_transaction_analysis.rs

```diff
@@ -42,12 +42,14 @@ impl SargonOS {
         instructions: String,
         blobs: Blobs,
         nonce: Nonce,
+        notary_public_key: PublicKey,
```

Comment:
NotaryPublicKey in the context of Pre-Auth is required for enclosed manifest analysis. It could be potentially generated internally in Sargon, but opted to expose through uniffi API so it is consistent with transaction preview.